src/emails/functions/reduceSanitizedDataPoints.ts

```diff
@@ -0,0 +1,13 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+// data is inteferred from the SanitizedDataPoints type, but we ran into inference issues directly importing it
+export function calculateSanitizedDataPoints(
```

Comment:
Nit, but I'd call this something like `sumSanitizedDataPoints` or `countSanitizedDataPoints` or something, to clarify what you're actually calculating.

And thanks for the comments - will be good in the future to know where this type came from, which otherwise we wouldn't have been able to deduce from anything here :)